boards/riscv/it8xxx2_evb/doc/index.rst

```diff
@@ -196,7 +225,10 @@ Ubuntu
       $ minicom -D /dev/ttyUSB0 -b 115200
 
 #. Open a second terminal window and use linux flash tool to flash your board.
-   You can find more details about tool usage from its document.
+
+   .. code-block:: console
+
+      $ sudo ~/itetool/ite -f build/zephyr/zephyr.bin
```

Comment:
Can add a new step 1 that instructs the user to download and compile the IT81202 Linux Flash Tool?

Reply:
The tar file contains a README file which instructs the user to compile and run the tool